packages/button/src/Button.tsx

```diff
@@ -331,8 +350,16 @@ const Button = React.forwardRef((props: ButtonProps, forwardRef) => {
     'size',
     'variant',
     'children',
+    'glyph',
   ]);
 
+  const modifiedGlyph =
+    glyph && children
+      ? React.cloneElement(glyph, {
+          className: cx({ [glyphMargins[size]]: glyph !== null }),
```

Comment:
It looks like `glyph` can be undefined, so we should use `glyph != null` instead to handle that case.